### PR TITLE
added h2 and p elements

### PR DESCRIPTION
**Pull request by necr0coder**
asd;lkajsdf;lkjasd

### CODE REVIEW

**Comment by necr0coder** [comment]
new comment

File: index.html [hunk 1]
```diff
@@ -7,5 +7,9 @@
   </head>
   <body>
     <h1>Hello world!</h1>
+    <h2>hello again</h2>
+    <p>blah</p>
+    <p>blah</p>
```
Comment (jrhurleycode) [inline index.html:12]: hey maybe change this

**Review by jrhurleycode — COMMENTED** [review]
looks good

File: index.html (hunk 1 above)
Comment (necr0coder) [reply index.html:12]: ok i will

**Review by jrhurleycode — APPROVED** [review]
looks good

**Pull request merged**